frontend/src/components/c-summary-charts.vue

```diff
@@ -479,31 +487,41 @@ export default {
       this.$store.commit('updateTabZoomInfo', info);
     },
 
-    getBaseTarget(target) {
+    getBaseTarget(target: HTMLElement | null): HTMLElement | null {
+      if (!target) {
+        // Should never reach here
```

Comment:
I think adding a log message could be more helpful here? Just in case the code enters here for whatever reason, we could trace the issue quickly.

Reply:
Agreed, just added a console statement & elaborated a bit more on the comment itself. The linter is currently setup to warn console statements, but I think that this rare false positive is fine as it can catch unintentional console statements from debugging that were accidentally committed.